sources/release-server/ReleaseServer.WebApi/Program.cs

```diff
@@ -1,15 +1,19 @@
+//--------------------------------------------------------------------------------------------------
+// <copyright file="Program.cs" company="Traeger IndustryComponents GmbH">
+//     This file is protected by Traeger Industry Components GmbH Copyright © 2019-2020.
+// </copyright>
+// <author>Timo Walter</author>
+//--------------------------------------------------------------------------------------------------
+
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 
 namespace ReleaseServer.WebApi
 {
-    public class Program
+    internal class Program
```

Comment:
Usually `Program` is marked as `public`.

Reply:
Done with https://github.com/Traeger-GmbH/release-server/pull/60/commits/2cb28c7104c2450252ad2f226c26b9bc0480c9a1